src/azure-cli/azure/cli/command_modules/synapse/manual/_params.py

```diff
@@ -226,6 +226,8 @@ def load_arguments(self, _):
                    arg_group=storage_arg_group,
                    help='The Storage Account Type.',
                    arg_type=get_enum_type(['GRS', 'LRS']))
+        c.argument('collation',
+                   help='Collation defines the rules that sort and compare data, and cannot be changed after SQL pool creation. The default collation is "SQL_Latin1_General_CP1_CI_AS".')
```

Comment:
Does this parameter has enum definition for allowed values?

Reply:
About sql server collation, there is many allowed values. There is a ms doc [Collation and Unicode support](https://docs.microsoft.com/en-us/sql/relational-databases/collations/collation-and-unicode-support?view=sql-server-ver16).  So I think it's hard to enum it.